src/classes/ ffTestStud.cls

```diff
@@ -0,0 +1,194 @@
+public with sharing class ffTestStud {
+
+  // Constructor.
+Public AccountDeDuper () {
+
+}
+public void badStuff() {}
+public void badStuff2() {}
+public void BADSTUFF3() {}
+public void BADSTUFF__4() {}
+
+Public List<Account> SearchAccount (Account pCurrentRecord, String pAccountName, String pPostCode, String pCity, id pRecordTypeId, Boolean pShowAllRecordTypes, Integer pSearchLimit) {
```

Comment:
* This method has a very long parameter list. I suggest to encapsulate many related parameters into a single object, or consider passing some parameters in the constructor prior to the invocation of this method.
* Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.